DeusEx/Classes/DXRFixup.uc

```diff
@@ -48,6 +48,19 @@ function CheckConfig()
         DecorationsOverwrites[i].explosionDamage = c.default.explosionDamage;
         DecorationsOverwrites[i].explosionRadius = c.default.explosionRadius;
         DecorationsOverwrites[i].bPushable = c.default.bPushable;
+
+        i++;
+        DecorationsOverwrites[i].type = "Van";
+        DecorationsOverwrites[i].bInvincible = false;
+        DecorationsOverwrites[i].HitPoints = 500;
+        DecorationsOverwrites[i].minDamageThreshold = 0;
+        c = class<DeusExDecoration>(GetClassFromString(DecorationsOverwrites[i].type, class'DeusExDecoration'));
+        DecorationsOverwrites[i].bFlammable = c.default.bFlammable;
+        DecorationsOverwrites[i].Flammability = c.default.Flammability;
+        DecorationsOverwrites[i].bExplosive = c.default.bExplosive;
+        DecorationsOverwrites[i].explosionDamage = c.default.explosionDamage;
+        DecorationsOverwrites[i].explosionRadius = c.default.explosionRadius;
+        DecorationsOverwrites[i].bPushable = c.default.bPushable;
```

Comment:
the van is nice for the hongkong garage area, if you get there early you can blow up the van to get to the tube that drops into versalife 2? maybe this should only be active with rearranged levels?